app/questionnaire/user_action_processor.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, schema, metadata):
         self.action = 'save_continue'
 
     def perform_action(self):
-        current_user.save()
+        pass
```

Comment:
Observation: Using this pattern for UserAction seems somewhat like overkill now there is only one concrete implementation for perform_action. Would a rest api for submit not suffice here?